src/reconstruction/comparison/modelBorgifier/src/TmodelFields.m

```diff
@@ -0,0 +1,82 @@
+% This file is published under Creative Commons BY-NC-SA.
+%
+% Please cite:
+% Sauls, J. T., & Buescher, J. M. (2014). Assimilating genome-scale 
+% metabolic reconstructions with modelBorgifier. Bioinformatics 
+% (Oxford, England), 30(7), 1036?8. http://doi.org/10.1093/bioinformatics/btt747
+%
+% Correspondance:
+% johntsauls@gmail.com
+%
+% Developed at:
+% BRAIN Aktiengesellschaft
+% Microbial Production Technologies Unit
+% Quantitative Biology and Sequencing Platform
+% Darmstaeter Str. 34-36
+% 64673 Zwingenberg, Germany
+% www.brain-biotech.de
+%
+function fields = TmodelFields(~)
+% TmodelFields returns the field names of the Tmodel structure. This is
+% to help allievate control between different scripts.
+%
+% USAGE:
+%    fields = TmodelFields(~)
+%
+%INPUTS:
+%    None
+% 
+% OUTPUTS:
+%    Fields:       A column cell array that contains the following in order. 
+%       rxnFields
+%       rNumField
+%       metFields
+%       mNumFields
+%       allFields
+%
+% CALLS:
+%    None
+% 
+% CALLED BY:
+%    cleanTmodel
+%    readCbTmodel
+%    verifyModel
+%    organizeModelCool
+%    TmodelStats
+%    orderModelFields
+%
+
+% Declare fields
+fields = cell(4,1) ;
+
+% Reaction related fields that are cell arrays.
+rxnFields = {'rxns' 'rxnID' 'rxnNames' 'subSystems' 'rxnECNumbers' ...
```

Comment:
maybe retrieve these fields from the getDefinedFieldProperties function, and potentially update the COBRAFieldDefinitions xls file to also contain rxnSEEDID